### DISCUSSION TITLE
How to overlay the observed data into the concentration-time curve of population simulation

### DISCUSSION THREAD

**Discussion by dat994** [Open-Systems-Pharmacology/Forum#1850] [Questions & Problems]
Dear all, 
Can I ask how to overlay the observed data in the concentration-time profile in of the population simulation? It is possible in individual simulation but I can not find a way to overlay it with population simulation. 
Many thanks

**Comment by StephanSchaller** [comment]
You cannot drag&drop the data into the plot?
